source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+function getGithubIoLink(): string {
+	return `${getRepositoryInfo().owner!}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const repos = select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]');
+	for (const repo of repos) {
+		const path = new URL(repo.href).pathname.slice(1);
+		const repoInfo = getRepositoryInfo(path);
+
+		repo.parentElement!.append(
+			<a
+				href={`https://${repoInfo.name!}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				<LinkExternalIcon/>
+			</a>
+		);
+	}
+}
+
+async function githubIoRepoInit(): Promise<void | false> {
+	const githubIoLink = getGithubIoLink();
+
+	if (select.exists('.octicon-link', await elementReady('.BorderGrid-cell'))) {
+		return false;
+	}
+
+	select('.BorderGrid-cell')!.append(
+		<div className="mt-3 d-flex flex-items-center">
+			<LinkIcon className="mr-2"/>
+			<span className="flex-auto min-width-0 css-truncate css-truncate-target width-fit">
+				<a
+					role="link"
+					target="_blank"
+					className="text-bold"
+					rel="noopener noreferrer"
+					href={`https://${githubIoLink}`}
+				>
+					{githubIoLink}
+				</a>
+			</span>
+		</div>
+	);
+}
+
+void features.add({
+	id: __filebasename,
+	description: 'Add a link to quickly visit the user’s github.io',
+	screenshot: false
```

Comment:
Don't add `screenshot: false` if you haven't added a screenshot yet. Let it fail until the feature is ready to have a screenshot.

```suggestion
```